jekyll/_cci2/testing-orbs.md

```diff
@@ -1,187 +1,132 @@
+        docker:
+          - image: cimg/base:stable
+        steps:
+          - my_command
 
-This testing is best completed by either publishing the orb as a dev version and then using it in your config and processing that config, or by using config packing to make it an inline orb and then processing that. Then you can use `circleci config` process and compare it to the expanded state you expect.
+workflows:
+  main:
+    jobs:
+      - test-command
 
-Another form of expansion testing can be done on the orb itself when your orb references other orbs. You can use the `circleci orb` process to resolve the orbs-dependent orbs and see how it would be expanded when you publish it to the registry.
+```
 
-Keep in mind that when testing expansion it's best to test the underlying structure of the data rather than the literal string that is expanded. A useful tool to make assertions about things in yaml is `yq`. This allows you to inspect specific structural elements rather than having fragile tests that depend on string comparisons or every aspect of an expanded job.
+For more information see the [Writing Inline Orbs]({{site.baseurl}}/2.0/reusing-config/#writing-inline-orbs) guide.
 
-The following steps illustrate doing basic expansion testing from the CLI:
+## Schema Validation
 
-1) Assume a simple orb in `src/orb.yml`
+To test whether an orb is valid YAML and is well-formed according to the CircleCI schema, use the CircleCI CLI command `circleci orb validate`.
 
-{% raw %}
-```yaml
-version: 2.1
+For example,given an orb with source at `./src/orb.yml`, run `circleci orb validate ./src/orb.yml` to receive feedback on whether the orb is valid and will pass through config processing. If there is an error, you will receive the first schema validation error encountered. Alternatively, you can pass STDIN rather than a file path.
 
-executors:
-  default:
-    parameters:
-      tag:
-        type: string
-        default: "curl-browsers"
-      docker:
-        - image:  circleci/buildpack-deps:parameters.tag
-
-jobs:
-  hello-build:
-    executor: default
-    steps:
-      - run: echo "Hello, build!"
-```
-{% endraw %}
+## Integration Testing
 
-2) Validate the orb with `circleci orb validate src/orb.yml`.
+Using [development orbs]({{site.baseurl}}/2.0/orb-concepts/#development-orbs), it is possible to publish changes to a live development version of an orb and use that development version within a live configuration for testing. Development versions of orbs can later be promoted to production orbs.
 
-3) Publish a dev version with `circleci orb publish src/orb.yml namespace/orb@dev:0.0.1`.
+Integrations tests validate the usage of your orb and its parameters in a live environment on CircleCI using sample project code (if required). Integration test jobs should be declared that both use the commands defined within your orb, and validate the correct functioning of those commands. See the following examples for testing a command, and testing a job.
```

Comment:
"Integration test jobs should be declared **within the live project environment** that both use the commands..." maybe add to clarify where these tests should be written?

Reply:
^ Ideally in the end they will be using the orb init command but I can't add much information about that yet.

Reply:
👍